### ISSUE TITLE
Is ws.send thread safe?I want to call it in sub thread

### ISSUE THREAD

**Issue by seetimee** [miguelgrinberg/flask-sock#85]

**Comment by miguelgrinberg** [comment]
Yes, you can pass the `ws` object to another thread and use it there. But if you plan to use the same `ws` object from multiple threads that are running at the same time then the answer is more complicated, I'd say that is probably best avoided or protected with a lock to ensure concurrent writes cannot interfere with each other.
